### DISCUSSION TITLE
Meta data value is altered.

### DISCUSSION THREAD

**Discussion by sanjaypaudel35** [zitadel/zitadel#7179] [Q&A]
I was inserting meta data value of the user i.e dob = "1997-07-16".
But when i fetch the userinfo from the userinfo api it returns dob value as 1997-07-w

#
![MicrosoftTeams-image (9)](https://github.com/zitadel/zitadel/assets/42547119/535f25eb-7094-41c3-b6bb-c54486a0e57b)

**Comment by adlerhurst** [comment]
Thanks for your report we have a look at why that happened
